code/userprog/addrspace.cc

```diff
@@ -136,7 +136,8 @@ AddrSpace::AddrSpace(OpenFile *exec) : fileTable(MaxOpenFiles) {
     
     baseDataSize = divRoundUp(size, PageSize) +  divRoundUp(UserStackSize,PageSize);//(old numPages w/1 stack) something to use publicly to find my stack
     //TESTING JACK REMOVED * 50 in front of divroundup below
-    numPages = divRoundUp(size, PageSize) +  50*divRoundUp(UserStackSize,PageSize);
```

Comment:
Keep this. this is for stack tracking PT. change numPhysPages to 32 in machine/machine.h